src/main/java/pages/AbstractPage.java

```diff
@@ -36,5 +40,10 @@ protected boolean waitForElementIsPresent(WebElement element) {
 
         return element.isEnabled();
     }
+
+    protected List<WebElement> findElements(By by) {
```

Comment:
please remove unused method

Reply:
Deleted, thank you for the pointing.
Please see in the following commit.